docs/man/docker-build.1.md

```diff
@@ -14,6 +14,11 @@ docker-build - Build a new image from the source code at PATH
 [**-q**|**--quiet**[=*false*]]
 [**--rm**[=*true*]]
 [**-t**|**--tag**[=*TAG*]]
+[**-m**|**--memory**[=*MEMORY*]]
+[**--memory-swap**[=*MEMORY-SWAP]]
+[**-c**|**--cpu-shares**[=*0*]]
```

Comment:
hmmm... that looks like a GH rendering issue, looks fine in editor.
This is a copy/paste from `docker-run.md`